src/clients/FlexbaseClient.Person.ts

```diff
@@ -134,4 +154,25 @@ export class FlexbaseClientPerson extends FlexbaseClientBase {
             return null;
         }
     }
+
+    async getAuthenticatedUserData(): Promise<PersonResponse> {
+        try {
+            const result =  await this.client.url('/user/self').get().json<Person>();
+            return {
```

Comment:
Same here

Reply:
This endpoint doesn't return a `success`